examples/deployment/deployment.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.12.29"
+  required_version = ">= 0.12"
```

Comment:
This was set to `">= 0.12.29"` because there are some changes which were fixed in that version as far as I remember. We should probably keep it like it was :) If this is confusing people then maybe you can bump it to `>=0.13` ?

Reply:
Interesting, I wasn't aware that we had issues in <0.12.29; 
does the provider not work at all under that version?

I'd just want us to not create too strong of a "support" message here. It may also be that the first thing we do when people see things not working properly on <0.12.29 is to upgrade to the latest 0.12.X

Just a thought. Anyways, i'd be happy to know what sort of issues were seen in earlier versions so that I have more context.

Reply:
Yeah, we need `0.12.29` for terraform to not crash when `terraform` block is set.